internal/compiler/widgets/material/scrollview.slint

```diff
@@ -117,6 +118,7 @@ export component ScrollView {
     flickable := Flickable {
         x: 0;
         y: 0;
+        interactive: false;
```

Comment:
This defaults to false because on android we should default to draggable.

Reply:
I'm a bit confused, wouldn't I want `interactive: true;` to default to draggable? And, does this also apply for Cupertino?

Reply:
right sorry my comment was not clear.
It currently defaults to true for the material style (as opposed to all the other style which explicitly set `interactive: false`

The material style is different because this is a style which is primarily interacted with touch because of android.
While the other style are to be used on desktop.

It is true that cupertino is currently the default on iOS, but our iOS support is not finished yet.